x-pack/plugins/security_solution/server/endpoint/routes/actions/isolation.ts

```diff
@@ -100,16 +109,6 @@ export const isolationRequestHandler = function (
   SecuritySolutionRequestHandlerContext
 > {
   return async (context, req, res) => {
-    // only allow admin users
-    const user = endpointContext.service.security?.authc.getCurrentUser(req);
-    if (!userCanIsolate(user?.roles)) {
-      return res.forbidden({
-        body: {
-          message: 'You do not have permission to perform this action',
-        },
-      });
-    }
```

Comment:
This is unnecessary now that we've the privileges added to the route as tags. Although, we might still need this to be communicated to the UI somehow as not having the privilege will not allow the route handler to execute.